src/main/java/org/example/gamePlay/GamePlay.java

```diff
@@ -0,0 +1,192 @@
+package org.example.gamePlay;
+
+
+import org.example.model.Card;
+import org.example.model.House;
+import org.example.model.Player;
+
+import java.util.Scanner;
+
+public class GamePlay {
+
+    private final DeckService deckService = new DeckService();
+    private final Validators validators = new Validators(new Scanner(System.in));
+    private final Player player;
+    private final House house;
+    private final Scanner scanner = new Scanner(System.in);
+    private int wager = 0;
+    private boolean wouldYouPlay = true;
+    private int remainingCards = 0;
+
+    public GamePlay(House house, Player player) {
+        this.house = house;
+        this.player = player;
+    }
+
+    public void start() {
+        String deckId = startNewGame();
+        while (wouldYouPlay && player.getPlayerPoints() > 0) {
+            if (remainingCards < 2) {
+                deckId = ifNotEnoughCardInDeck();
+            }
+            promptForNewGame(deckId);
+        }
+    }
+
+    private String startNewGame() {
+        player.setPlayerPoints(100);
+        System.out.println(" Player otrzymuje 100 punktow ! ");
+        wouldYouPlay = true;
```

Comment:
wouldYouPlay already set to true

Reply:
I forgot remove, after testing part of code.Fixed.